<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pass</class>
 <widget class="QDialog" name="pass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="yes">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>191</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button9">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button10">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
